### ISSUE TITLE
if "aff_loss_edge" and "aff_loss_nedge" do not have a coefficient of "dec" that decreases with "step", can "aff_loss" converge?

### ISSUE THREAD

**Issue by ef-ever** [twke18/Adaptive_Affinity_Fields#6]
Excuse me, if "aff_loss_edge" and "aff_loss_nedge" do not have a coefficient of "dec" that decreases with "step", can "aff_loss" converge? When I run "train_affinity.py" with the parameters mentioned in "train_pspnet_affinity.sh", "aff_loss_edge" will not fall until 2.88.

**Comment by twke18** [comment]
Yes, according to our empirical experience, it is necessary to exponentially decrease the affinity field loss during training.
